File: Pyramid Pattern/code10.c
#include <stdio.h>

void main (){

	int row;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0; i < row ; i++){

		for(int a = 0; a < i;a++ ){

			printf(" \t");
		}
		for(int j = 0; j < ((row-i)*2)-1;j++){

			printf("%c\t",65+i);

		}
		printf("\n\n");

	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/9/code10.java

import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s= new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		int temp = row*row + row;
		for(int i = 0; i < row ; i++){
			
			for(int j = 0; j < row ; j++){
					
				System.out.print(temp+"		");
				temp--;
				
			}

				System.out.print("\n\n");

		}

	}

}
